examples/tfengine/generated/team/project_apps/main.tf

```diff
@@ -77,23 +77,6 @@ resource "google_binary_authorization_policy" "policy" {
     enforcement_mode = "ENFORCED_BLOCK_AND_AUDIT_LOG"
   }
 
-  # Recommendations from https://cloud.google.com/binary-authorization/docs/policy-yaml-reference#admissionwhitelistpatterns
-  admission_whitelist_patterns {
-    name_pattern = "gcr.io/google_containers/*"
```

Comment:
I don't see this one listed as those covered under https://cloud.google.com/binary-authorization/docs/policy-yaml-reference#globalpolicyevaluationmode. Should we keep it?